source/blog/_posts/2016-09-09-99-paper-cuts-status.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Status of the 99 papercuts project
+github_link: blog/_posts/2016-09-09-99-paper-cuts-status.md
+authors: [bc]
+---
+
+We are at day five of our papercuts project, ten days to go. 
+
+Continuing until Monday, 19th September you can participate in our [99 Papercuts Project](https://developers.shopware.com/99-paper-cuts/) and make Shopware better - one papercut at a time.
+
+The feedback so far is pretty good, we receive a lot of contributions and [merged](https://github.com/shopware/shopware/pulls?q=is%3Apr+%5BPapercut%5D+is%3Aclosed) a good bunch of pull requests.
+Also we notice increased activity in our [IRC Channel](/contributing/irc/) where you can get in touch with the community.
+
+You can help by reviewing the [open](https://github.com/shopware/shopware/pulls?utf8=%E2%9C%93&q=%3Apr%20is%3Aopen%20%5BPapercut%5D) pull requests and provide feedback.
+
+## New Pull Request Workflow
+
+If you are a regular contributor to Shopware you may have noticed that we changed the pull request workflow for the time of the papercuts project.
+
+Normally, when you open a pull request we will first schedule it, that means a script fetches the pull requests information and creates a ticket in our issue tracker. In one of our next scrum sprint the team will then put tickets with the `scheduled` label into the sprint and work on it.
```

Comment:
Normally, when you open a pull request we will first schedule it. That means a script fetches the pull request's information and creates a ticket in our Issue Tracker. In one of our next scrum sprints, the team will place the tickets with the status "scheduled" into the sprint and work on it.